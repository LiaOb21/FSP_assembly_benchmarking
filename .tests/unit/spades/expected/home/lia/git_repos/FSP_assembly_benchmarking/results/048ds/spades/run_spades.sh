set -e
true
true
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/spades-core /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/config.info /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/careful_mode.info
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/spades-core /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/config.info /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/careful_mode.info
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/spades-core /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/config.info /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/careful_mode.info
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3 /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/copy_files.py /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/before_rr.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/before_rr.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_after_simplification.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_after_simplification.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/first_pe_contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/first_pe_contigs.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/strain_graph.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/strain_graph.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.paths /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.paths /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_with_scaffolds.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_with_scaffolds.gfa /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph.fastg /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph.fastg /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.paths /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.paths
true
true
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3 /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/correction_iteration_script.py --corrected /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta --assembled /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc/assembled_contigs.fasta --assembly_type contigs --output_dir /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades --bin_home /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3 /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/correction_iteration_script.py --corrected /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta --assembled /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc/assembled_scaffolds.fasta --assembly_type scaffolds --output_dir /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades --bin_home /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin
true
/home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3 /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py --result_scaffolds_filename /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta --misc_dir /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc --threshold_for_breaking_scaffolds 3
true
