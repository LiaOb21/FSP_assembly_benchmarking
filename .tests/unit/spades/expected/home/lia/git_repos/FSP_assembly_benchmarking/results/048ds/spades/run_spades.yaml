- STAGE: Before start
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Before_start_859XYDGZFFSLBPKO8DMJJ4LUD63X75SO
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: before_start
- STAGE: Assembling
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Assembling_OMBGNNSLWGQ59156L9MT8AEIWUB7G0N5
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: as_start
- STAGE: K21
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/config.info
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K21/configs/careful_mode.info
  config_dir: K21
  del_after: []
  job_uuid: hpcSPAdes_K21_B3HN7RODL0GV80A5ED00F6J4CUHL5YER
  mpi_support: true
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/{spades_core}
  short_name: k21
- STAGE: K33
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/config.info
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K33/configs/careful_mode.info
  config_dir: K33
  del_after: []
  job_uuid: hpcSPAdes_K33_0BDIMN3VZK9KMH9IO80GSPQQFYP4ZF2C
  mpi_support: true
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/{spades_core}
  short_name: k33
- STAGE: K55
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/config.info
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/configs/careful_mode.info
  config_dir: K55
  del_after: []
  job_uuid: hpcSPAdes_K55_OO1QK4AUVASH0G3Y94NFE1UWRX9PC7JI
  mpi_support: true
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/{spades_core}
  short_name: k55
- STAGE: Copy files
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/copy_files.py
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/before_rr.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/before_rr.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_after_simplification.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_after_simplification.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/first_pe_contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/first_pe_contigs.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/strain_graph.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/strain_graph.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/scaffolds.paths
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.paths
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph_with_scaffolds.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph_with_scaffolds.gfa
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/assembly_graph.fastg
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/assembly_graph.fastg
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/K55/final_contigs.paths
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.paths
  config_dir: ''
  del_after:
  - .bin_reads
  - tmp/spades_4s6mt7qr
  job_uuid: SPAdes_Copy_files_JCFYE7KCAZ0EWMWPJJCJ8J4GXLXOPEUT
  mpi_support: false
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3
  short_name: copy_files
- STAGE: Assembling
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Assembling_ZOJIJN67NU9QL9N9CX7IHERN28V0USXS
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: as_finish
- STAGE: Mismatch correction
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Mismatch_correction_1BW9CF0NSN65M2TJ7O180THAASC5JLOS
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: mc_start
- STAGE: Mismatch correction contigs
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/correction_iteration_script.py
  - --corrected
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/contigs.fasta
  - --assembled
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc/assembled_contigs.fasta
  - --assembly_type
  - contigs
  - --output_dir
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades
  - --bin_home
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin
  config_dir: mismatch_corrector/contigs
  del_after:
  - mismatch_corrector/contigs/tmp
  - tmp/corrector_5nsrlv8e
  job_uuid: SPAdes_Mismatch_correction_contigs_UU5LV1X3KOUN7ZOQ0A0NU158M5DVLGIE
  mpi_support: false
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3
  short_name: mc_contigs
- STAGE: Mismatch correction scaffolds
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/correction_iteration_script.py
  - --corrected
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta
  - --assembled
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc/assembled_scaffolds.fasta
  - --assembly_type
  - scaffolds
  - --output_dir
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades
  - --bin_home
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin
  config_dir: mismatch_corrector/scaffolds
  del_after:
  - mismatch_corrector/scaffolds/tmp
  - tmp/corrector_subs8ksl
  job_uuid: SPAdes_Mismatch_correction_scaffolds_CTH52IV0ITAQW5K3SMBXO525Y45AGCYV
  mpi_support: false
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3
  short_name: mc_scaffolds
- STAGE: Mismatch correction
  args: []
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Mismatch_correction_G24HPZLH7RRXW5L1QSHHOSY2SSD72VVE
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: mc_finish
- STAGE: Breaking scaffolds
  args:
  - /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py
  - --result_scaffolds_filename
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/scaffolds.fasta
  - --misc_dir
  - /home/lia/git_repos/FSP_assembly_benchmarking/results/048ds/spades/misc
  - --threshold_for_breaking_scaffolds
  - '3'
  config_dir: ''
  del_after: []
  job_uuid: SPAdes_Breaking_scaffolds_PGEPRYQM6VL9F4THJTY7E9TALMKCM1LJ
  mpi_support: false
  output_files: []
  path: /home/lia/git_repos/FSP_assembly_benchmarking/.snakemake/conda/4cb5632bbd0b297e85e95fa7b04bc60c_/bin/python3
  short_name: bs
- STAGE: Terminate
  args: []
  config_dir: ''
  del_after:
  - configs
  job_uuid: SPAdes_Terminate_8D7TZPJ2MPOLV9VAGKZNO579RILQMK3T
  mpi_support: false
  output_files: []
  path: 'true'
  short_name: terminate
